docs/usage/index.md

```diff
@@ -73,40 +77,30 @@ Starting with Pyodide 0.25.0, Node.js < 18 is no longer officially supported.
 Older versions of Node.js might still work, but they are not tested or guaranteed to work.
 ```
 
-```{note}
-The following instructions have been tested with Node.js 18.5.0. To use
-Pyodide with older versions of Node, you might need to use  additional command line
-arguments, see below.
-```
```

Comment:
I think this note is superseded by the note above it saying that older versions aren't supported.